packages/tools/pie-icons-react/scripts/react-icons-gen.js

```diff
@@ -14,34 +14,57 @@ async function checkDirExists (directoryPath) {
         await fs.ensureDir(directoryPath);
         console.log(`Directory "${directoryPath}" exists.`);
     } catch (err) {
-        console.error(err)
+        console.error(err);
     }
 }
 
-const handleComponentName = name => name.replace(/\-(\d+)/, '$1'); // eslint-disable-line no-useless-escape
+const handleComponentName = (name) => name.replace(/\-(\d+)/, '$1'); // eslint-disable-line no-useless-escape
 
 // check that the /icons directory exists, if not create it
 checkDirExists(ICONS_DIR);
 
 // open a write stream to index.tsx
 const indexFile = fs.createWriteStream(
     filePath,
-    err => {
+    (err) => {
         console.error(err);
-    }
+    },
```

Comment:
is this trailing comma intentional?

Reply:
I'm not sure if this is a linting change or me 😂 just tried to remove so we'll see

Reply:
It's a linting thing – it's part of the new ESLint ruleset to add trailing commas